docs/specs/forecast-api.md

```diff
@@ -122,8 +120,8 @@ generated using the nearest forecast time.
 
 ### flight level
 
-The fl value represents the flight level (*hectofeet*) of the returned
-netCDF global grid.
+The `{fl}` value represents the flight level (*hectofeet*) of the returned
```

Comment:
Could you rename this to `flight_level`? This would match both the result coordinate name and our parameter/coordinate name.

Reply:
You mean the param name, right? If so, I think we're already good (see signature here):
https://github.com/contrailcirrus/contrail-forecast/blob/refactor/v1-signature/docs/specs/forecast-api.md#signature-1

The bracketed fields just represent the parameter value passed.
We're just using shorthand for brevity, but if you'd prefer that the signature read:
`?flight_level={flight_level}&time={time}` etc... instead of `?flight_level={fl}&time={ts}`, I'm OK with that.

Reply:
This was an oversight from my end – the short form for the placeholder is perfectly fine! Thanks for pointing it out :)